config/default/manager_auth_proxy_patch.yaml

```diff
@@ -0,0 +1,25 @@
+# This patch inject a sidecar container which is a HTTP proxy for the 
+# controller manager, it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
```

Comment:
Any idea what this thing does? Seems like we just picked up another image for our product :(

Reply:
If we want to start including that then yeah. I don't think it is necessary yet.